cluster-up/cluster/kind/common.sh

```diff
@@ -141,6 +141,35 @@ function _fix_node_labels() {
     done
 }
 
+function _yq() {
```

Comment:
From where this `_` convention is coming from?

Reply:
I noticed that on this file function that being used only in that file starts with `_`